deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -635,12 +649,10 @@
     `with-id?`: If `with-id?` equals to true, all the referenced pages will have new db ids.
     `format`: content's format, it could be either :markdown or :org-mode.
     `options`: Options supported are :user-config, :block-pattern :supported-formats,
-               :extract-macros, :extracted-block-ids, :date-formatter, :page-name and :db"
-  [blocks content with-id? format {:keys [user-config db page-name extracted-block-ids] :as options}]
```

Comment:
moved to https://github.com/logseq/logseq/pull/8823/files#diff-a1e2a20e287e81626ce5ec90828dbc2d0d6c789401d44eeb842b5ffe0185038aR167